lib/manageiq_performance/reporter.rb

```diff
@@ -142,8 +142,7 @@ def column_size_for header, request_id
       @column_size_for[request_id][header] ||=
         (HEADERS[header].map { |header_column|
           @report_data[request_id][header_column].map {|i|
-            value = i.to_s
-            value = "%.1f" % value unless value.class.ancestors.include?(Integer)
+            value = (i && !i.class.ancestors.include?(Integer)) ? "%.1f" % i : i.to_s
```

Comment:
Oh, now I remember why the `.to_s` is there.  If we did a `size` on a number, that wouldn't make sense, we are trying to get the character length (this is why this same code exists also exists in line 127, but doesn't need the `.to_s`).

A few things with this:

* I think some of this might be could be solved with https://github.com/ManageIQ/manageiq-performance/pull/17 .  Try running:
  
  ```console
  $ git apply <(curl -L https://github.com/ManageIQ/manageiq-performance/pull/17.patch)
  ```
  
  On master, and run `miqperf report` again to see if the errors go away.  I think I covered some of the same issues there that might avoid some of this.
* A change here might still be necessary, but might I suggest doing one of the following instead of a ternary:
  - Swap the `.to_s` line to happen after the `include?` line, or even just appended it right before the `.size` call in the next line
  - Change the line to `value = "%.1f" % i unless i.class.ancestors.include?(Integer)` (`value` => `i` on the right hand side of the equation)

The ternary just clutters up the code, and the `.to_s` can be applied regardless if it is a string or a number, so no reason to make that conditional (pretty sure `.to_s` on a string is a no-op anyway).